uitools/toolkitcpp/docs/NorthArrow.md

```diff
@@ -0,0 +1,9 @@
+[![ArcGIS Maps SDK for Qt](https://img.shields.io/badge/ArcGIS%20Maps%20SDK%20for%20Qt-0b5394)](https://developers.arcgis.com/qt/) [![ArcGIS Maps SDK for Qt toolkit](https://img.shields.io/badge/ArcGIS%20Maps%20SDK%20for%20Qt%20toolkit-ea4d13)](https://github.com/Esri/arcgis-maps-sdk-toolkit-qt) [![Qt Quick UI components](https://img.shields.io/badge/Qt%20Qt%20Quick%20UI%20components-ea4d13)](../../toolkitcpp/)
+
+# NorthArrow - Qt Quick UI/QML Type
+
+The [NorthArrow - Qt Quick UI/QML Type](https://developers.arcgis.com/qt/toolkit/api-reference/qml-northarrow.html) displays a compass overlaid on the GeoView, with the compass heading matching the current rotation of the MapView, or Camera heading of the SceneView. Double clicking on the NorthArrow triggers the heading of the connected GeoView to be orientainted to 0.
+
+NOTE: default width and height is 48 pixels.
```

Comment:
```suggestion
NOTE: default width and height is 48 pixels. It is exaggerated in the example below to highlight its behavior.
```

This is optional, but I still feel users will see this and think it will be that large by default.

Reply:
Sure thing @JamesMBallard . I will accept the auto-commit suggestion.